### PR TITLE
import WCSTable, fixed some syntax and docstrings

### PR DESCRIPTION
**Pull request by glentner**
I didn't add anything, but fixed some two things that were/would cause exceptions.

### CODE REVIEW

File: astrospec/__init__.py [hunk 1]
```diff
@@ -4,6 +4,15 @@
 This is an Astropy affiliated package.
 """
 
+# Elevate objects to the package level
+from .spectrum import Spectrum1D
```
Comment (hamogu) [inline astrospec/__init__.py]: This should be `from .spectrum1d import Spectrum1D` (or you rename the `spectrum1d.py` file.

File: astrospec/spectrum1d.py [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
+
 import numpy as np
-from scipy inport interpolate
+from scipy import interpolate
```
Comment (hamogu) [inline astrospec/spectrum1d.py:5]: astropy does not have scipy as a dependency, but I think we can do that. Do you agree?

File: astrospec/spectrum1d.py [hunk 3]
```diff
@@ -8,10 +9,18 @@
...
-    """A 1 dimensionan spectrum.
-    
+    """A 1-dimensional spectrum.
+
+    Attributes
+    ----------
+
+    Methoods
+    --------
+
```
Comment (hamogu) [inline astrospec/spectrum1d.py]: I don't see a reason to list the attributes and methods here. You can always get them with `spectrum1d.<tab>` in IPython, with `dir(spectrum1d)` or in the sphinx docs where we auto-generate a list of all methods (and those attributes that have a sphinx compatible docstring).

File: docs/install.rst [hunk 4]
```diff
@@ -12,7 +12,7 @@ Astrospec has the following requirements:
 
 One easy way to get these dependencies is to install a python distribution like `anaconda <http://continuum.io/>`_.
 
-Installing psfsubtraction
+Installing astrospec
 =========================
```
Comment (hamogu) [inline docs/install.rst]: Good catch!

File: astrospec/__init__.py (hunk 1 above)
Comment (glentner) [inline astrospec/__init__.py]: Yes, I missed that. It should be `from .spectrum1d import Spectrum1D`.

File: astrospec/spectrum1d.py (hunk 2 above)
Comment (glentner) [inline astrospec/spectrum1d.py:5]: Yes, I think scipy being a dependency is necessary, not just for interpolation.

File: astrospec/spectrum1d.py (hunk 3 above)
Comment (glentner) [inline astrospec/spectrum1d.py]: We should decide this now I suppose. Astropy is rather inconsistent with this. I prefer numpy style doc strings to google style. You are correct that introspection gives us what we need in the IPython shell and elsewhere. We need not list "Methods" (I can't believe I miss spelled that). But I think it is helpful to list Attributes (the major ones that are important to the class. I know you can tab complete to get everything that Spectrum1D.\* is possible, but I think it's still a good idea to briefly state the major attributes and what they are for.

Comment (hamogu) [inline astrospec/spectrum1d.py]: Then I suggest that we add those when we have them. It's not clear to me yet what attribute we would document here.
The WCS maybe? That requires a full chapter...

File: astrospec/__init__.py (hunk 1 above)
Comment (hamogu) [inline astrospec/__init__.py]: Do you want to fix that and push a new commit?

File: docs/install.rst (hunk 4 above)
Comment (glentner) [inline docs/install.rst]: I found a few more of these type of thing in this file, referencing 'ccdproc'

**Comment by glentner** [comment]
Shall we proceed then?

**Comment by hamogu** [comment]
Let's give Travis some time to pass or fail...

**Comment by hamogu** [comment]
I also noticed that you do your work on your "master" branch. That might work in the beginning, but it will be very confusing later and makes for a little convoluted git commit history. The reason is this:
If I now merge on github and then you pull it should be all fine. But If I merge on github and then I add something to that while you also work on a feature then the "master" on github will differ from your master. So, if you "git pull" to get my changes, then you have a local merge commit and then you push that back up to a pull request, so that your branch then has the git master, my feature and your feature and a merge. That's going to be different from the "master" on github" and merging your next pull request will generate a merge that contains your changes and my changes again...
The code _should_ be fine, we we get a lot of merges and re-merges and potential merge conflicts.

If more than one person works on a repro, it's usually safer to do new things in new branches and merge them only once.

**Comment by glentner** [comment]
Yes, that's the best way to do things (regarding feature-branches). I merged your pull-request and then made a trivial change to the README.rst file and  pushed that up again.

**Pull request merged**